easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.0.eb

```diff
@@ -11,7 +11,10 @@ description = """The OpenSSL Project is a collaborative effort to develop a robu
 toolchain = SYSTEM
 toolchainopts = {'pic': True}
 
-osdependencies = ['perl']
+osdependencies = [
+    ('pkg-config', 'pkgconfig'),
```

Comment:
@lexming How about a proper build dependency, with a pkg-config we build from source?

I would expect it to be unlikely we'll run into lots of trouble when building `pkg-config` with the OS compiler...

`perl` is pretty much always there, but that's not the case for `pkg-config` imho...

Reply:
as discussed, the default search paths of pkg-config in `pkg-config-0.29.2.eb` are limited to the installation directory. Hence, it does not see any of the `.pc` files in the host system. This is fine for all other pkg-config easyconfigs in custom toolchains, but in the case of pkg-config in `SYSTEM` we can add the `pkgconfig` directories in the host system to the default `pc_path`.

The proper way to retrieve the `pc_path` in the system is with `pkg-config` itself and the command
```
pkg-config --variable pc_path pkg-config
```
Since that would require having and `os_dependency` on pkg-config in `pkg-config-0.29.2.eb`, I resorted to just use `find` on existing `pkgconfig` dirs. This should effectively generate the same result.